tests/unit/compiler/venom/test_multi_entry_block.py

```diff
@@ -132,7 +132,7 @@ def test_multi_entry_block_with_dynamic_jump():
     assert ctx.normalized, "CFG should be normalized"
 
     finish_bb = ctx.get_basic_block(finish_label.value)
-    cfg_in = list(finish_bb.cfg_in.keys())
+    cfg_in = list(finish_bb.cfg_in)
     assert cfg_in[0].label.value == "target", "Should contain target"
-    assert cfg_in[1].label.value == "finish_split___global", "Should contain finish_split___global"
-    assert cfg_in[2].label.value == "finish_split_block_1", "Should contain finish_split_block_1"
+    assert cfg_in[1].label.value == "__global_split_finish", "Should contain __global_split_finish"
```

Comment:
i'm not a huge fan of the hardcoded labels (i'd prefer to test properties on the cfg), but it's ok for now